include/seqan/align/dp_formula_affine.h

```diff
@@ -255,7 +236,6 @@ _internalComputeScore(DPCell_<TScoreValue, AffineGaps> & activeCell,
     TScoreValue cmpG = cmpGt(rightCompare, activeCell._horizontalScore);
     TScoreValue cmpE = cmpEq(rightCompare, activeCell._horizontalScore);
     activeCell._horizontalScore = blend(activeCell._horizontalScore, rightCompare, cmpG);
-    activeCell._score = activeCell._horizontalScore;
 
     TScoreValue result = leftTrace;
```

Comment:
And why do you need to initialize `result` with leftTrace, instead of directly blending it
`TWhateverType result = blend(leftTrace, rightTrace, cmpG);`